packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -223,7 +223,12 @@
 
 		<!-- Output section -->
 		<template #preview>
-			<tera-drilldown-section v-if="showOutputSection">
+			<tera-drilldown-section
+				:is-loading="isLoading"
+				:loading-progress="props.node.state.currentProgress"
+				:is-blank="!showOutputSection"
+				:blank-message="'Click \'Run\' to begin calibrating'"
+			>
```

Comment:
I think calibrate and ensemble-calibrate are a bit different, there is a loading spinner, but there is also the rendering of loss-values that gets rendered while the spinner is spinning. 

I am not sure if this covers this particular use case.